notebooks/lesson07_classification.ipynb

```diff
@@ -0,0 +1,3932 @@
+{
```

Comment:
<p>since it is a central concept, you could maybe repeat what these numbers mean? maybe rephrase it in terms of "if our model classifies a customer as churning it is x% of the time correct. if it says the customer is not churning it is y% of the time wrong." maybe mention that these are very central questions for example in drug development and medicine. maybe you could also show the unnormalized confusion matrix (otherwise it is hard to compare elements in columns).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='152'/>

Reply:
<p>thanks - included!</p>